tools/internal_ci/linux/pull_request/grpc_basictests_c_opt.cfg

```diff
@@ -26,5 +26,5 @@ action {
 
 env_vars {
   key: "RUN_TESTS_FLAGS"
-  value: "-f basictests linux c opt --inner_jobs 16 -j 1 --internal_ci --max_time=3600"
+  value: "-f basictests linux c opt --inner_jobs 16 -j 1 --internal_ci --max_time=3600 --bq_result_table tmp"
```

Comment:
Is this change intended?

Reply:
I'm creating a tmp BQ table to make sure the URL is being saved as intended. I'll remove this before merging.